test/test_runner.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var Metamocha = require('../lib/metamocha');
+
+// Instantiate new Metamocha
+var metamocha = new Metamocha();
+
+// Add files from directory
+metamocha.addFolder('test/');
+
+// Apply a filter
+metamocha.addFilter({
```

Comment:
If, as suggested in my previous comment, `Metamocha.addFilter()` took a function as its argument, this block could be shortened to:
```javascript
metamocha.addFilter(function(test) {
  return test.hasOwnProperty('metadata');
});
```